scripts/AbinsModules/SPowderSemiEmpiricalCalculator.py

```diff
@@ -367,6 +376,7 @@ def _calculate_s_powder_one_atom(self, atom=None):
                 s = self._calculate_s_powder_one_atom_core(atom=atom)
```

Comment:
Line 321 could be rewritten with a list comprehension `result = [self._s_calculate_power_one_atom(atom=atom) for atom in atoms]`